package.json

```diff
@@ -62,9 +73,14 @@
     "webpack-manifest-plugin": "1.3.2",
     "whatwg-fetch": "2.0.3"
   },
+  "peerDependencies": {
+    "patternfly-react": "^2.14.0",
+    "react": "^16.4.2",
+    "react-dom": "^16.4.2"
+  },
+  "dependencies": {},
   "scripts": {
```

Comment:
We can add `validations`, as @vojtechszocs proposed in a meeting.
Plus more explicit semver format for dependencies and update manually.

Reply:
:+1: but please do it on another PR that is specific for validations.

Reply:
Thanks for bringing this up. To summarize, validations are meant to be small programs (exporting a function that returns `true` or `false`) to check cross-cutting concerns of the project structure, like "each React component has associated test" and "we use a consistent dependency semver format".